pkg/helm/values/eks.go

```diff
@@ -0,0 +1,72 @@
+package values
+
+import (
+	"strings"
+
+	"github.com/loft-sh/vcluster/pkg/helm"
+	"github.com/loft-sh/vcluster/pkg/log"
+)
+
+var EKSAPIVersionMap = map[string]string{
+	"1.21": "public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.21.5-eks-1-21-8",
+	"1.20": "public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.20.11-eks-1-20-102",
+}
+
+var EKSControllerVersionMap = map[string]string{
+	"1.21": "public.ecr.aws/eks-distro/kubernetes/kube-controller-manager:v1.21.5-eks-1-21-8",
+	"1.20": "public.ecr.aws/eks-distro/kubernetes/kube-controller-manager:v1.20.11-eks-1-20-10",
+}
+
+var EKSEtcdVersionMap = map[string]string{
+	"1.21": "public.ecr.aws/eks-distro/etcd-io/etcd:v3.4.16-eks-1-21-8",
+	"1.20": "public.ecr.aws/eks-distro/etcd-io/etcd:v3.4.15-eks-1-20-10",
+}
+
+var EKSCoreDnsVersionMap = map[string]string{
+	"1.21": "public.ecr.aws/eks-distro/coredns/coredns:v1.8.4-eks-1-21-8",
+	"1.20": "public.ecr.aws/eks-distro/coredns/coredns:v1.8.3-eks-1-20-10",
+}
+
+func getDefaultEKSReleaseValues(chartOptions *helm.ChartOptions, log log.Logger) (string, error) {
+	serverVersionString := GetKubernetesVersion(chartOptions.KubernetesVersion)
+	serverMinorInt, err := GetKubernetesMinorVersion(chartOptions.KubernetesVersion)
+	if err != nil {
+		return "", err
+	}
+
+	apiImage := EKSAPIVersionMap[serverVersionString]
+	controllerImage := EKSControllerVersionMap[serverVersionString]
+	etcdImage, ok := EKSEtcdVersionMap[serverVersionString]
+	corednsImage, ok := EKSCoreDnsVersionMap[serverVersionString]
+	if !ok {
+		if serverMinorInt > 21 {
+			log.Infof("officially unsupported host server version %s, will fallback to virtual cluster version v1.23", serverVersionString)
+			apiImage = EKSAPIVersionMap["1.21"]
+			controllerImage = EKSControllerVersionMap["1.21"]
+			etcdImage = EKSEtcdVersionMap["1.21"]
+			corednsImage = EKSCoreDnsVersionMap["1.21"]
+		} else {
+			log.Infof("officially unsupported host server version %s, will fallback to virtual cluster version v1.20", serverVersionString)
+			apiImage = EKSAPIVersionMap["1.20"]
+			controllerImage = EKSControllerVersionMap["1.20"]
+			etcdImage = EKSEtcdVersionMap["1.20"]
+			corednsImage = EKSCoreDnsVersionMap["1.20"]
+		}
+	}
+
+	// build values
+	values := `api:
+	image: ##API_IMAGE##
+  controller:
+	image: ##CONTROLLER_IMAGE##
+  etcd:
+	image: ##ETCD_IMAGE##
+  coredns:
+	image: ##COREDNS_IMAGE##
+`
```

Comment:
This should look like:
```
api:
  image: ##API_IMAGE##
controller:
  image: ##CONTROLLER_IMAGE##
etcd:
  image: ##ETCD_IMAGE##
coredns:
  image: ##COREDNS_IMAGE##
```

But currently has wrong intendation:
```
api:
	image: ##API_IMAGE##
  controller:
	image: ##CONTROLLER_IMAGE##
  etcd:
	image: ##ETCD_IMAGE##
  coredns:
	image: ##COREDNS_IMAGE##
```
which will lead to problems deploying later on